org.eclipse.xtext.dev-bom/build.gradle

```diff
@@ -27,11 +27,11 @@ dependencies {
         api "org.eclipse.emf:org.eclipse.emf.ecore.change:2.14.0"
         api "org.eclipse.emf:org.eclipse.emf.ecore.xmi:2.16.0"
         api "org.eclipse.emf:org.eclipse.emf.ecore:2.20.0"
-        api "org.eclipse.emf:org.eclipse.emf.mwe.core:1.6.1"
-        api "org.eclipse.emf:org.eclipse.emf.mwe.utils:1.6.1"
-        api "org.eclipse.emf:org.eclipse.emf.mwe2.launch:2.12.1"
-        api "org.eclipse.emf:org.eclipse.emf.mwe2.lib:2.12.1"
-        api "org.eclipse.emf:org.eclipse.emf.mwe2.runtime:2.12.1"
+        api "org.eclipse.emf:org.eclipse.emf.mwe.core:[1.6.1,1.7.0)"
```

Comment:
am not sure if this is a good idea

Reply:
but it allows to use the beta in mwe without using fake milestone or snapshot here